.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
delete this file, no need to upload it [each one has their own in the working dir]

Reply:
it was uploaded automatically, deleted now